src/pages/Swap/index.tsx

```diff
@@ -398,6 +427,17 @@ const Swap = () => {
               )}
             </AutoColumn>
             <BottomGrouping>
+              {disableSwap && (
```

Comment:
Show descriptive text if button is disabled to direct user to V2
 
<img width="501" alt="Screenshot 2021-05-12 at 11 31 48" src="https://user-images.githubusercontent.com/79279477/117961120-a80c5e00-b315-11eb-8ae3-1e5a1ca10a15.png">